### PR TITLE
Chore: Add Address and contact deletion

### PR DESCRIPTION
**Pull request by TaniaSakun**
## Description:
1. Added method to delete contact 
2. Added address parameter to the controller

### CODE REVIEW

File: apps/cli/app/services.py [hunk 1]
```diff
@@ -100,15 +104,28 @@ def update_contact(prev: Contact) -> None:
...
         "Enter days duration from today",
         error_message="Duration should be an Integer",
         optional=True,
     )
-
-    day_amount = day_duration if type(day_duration) == int else 7
+    try:
+        day_amount = int(day_duration)
+    except:
+        day_amount = 7
```
Comment (iMykhailychenko) [inline apps/cli/app/services.py:128]: the library from where we get this prompt util is already has int type validation, will add it later

File: apps/cli/app/services.py [hunk 2]
```diff
@@ -45,7 +45,7 @@ def create_new_contact(*_) -> None:
     )
     payload = ContactPayload(name=name)
     result = controller(Actions.GET, payload)
-
+    console.print(result)
```
Comment (iMykhailychenko) [inline apps/cli/app/services.py:48]: do we need it here?

Comment (TaniaSakun) [reply apps/cli/app/services.py:48]: Of course, no 😅. I'll remove it

**Review by iMykhailychenko — APPROVED** [review]

**Comment by iMykhailychenko** [comment]
Let's merge it, I'll try to find what is the problem with delete in CLI app

**Pull request merged**